src/tests/containerizer/port_mapping_tests.cpp

```diff
@@ -2203,6 +2219,9 @@ TEST_F(PortMappingIsolatorTest, ROOT_ScaleIngressWithCPUAutoConfig)
   // Kill the container
   AWAIT_READY(launcher.get()->destroy(containerId1));
   AWAIT_READY(isolator.get()->cleanup(containerId1));
+
+  delete launcher.get();
+  delete isolator.get();
```

Comment:
there's some extra deletes in here, can you split this out into its own commit or PR?